scripts/publish_metadata.py

```diff
@@ -0,0 +1,27 @@
+from multibase import multibase
+from multicodec import multicodec
+from aea.helpers.cid import to_v1
+from aea_cli_ipfs.ipfs_utils import IPFSTool
+from scripts.generate_metadata import METADATA_FILE_PATH
+
+PREFIX = "f01701220"
+IPFS_PREFIX_LENGTH = 6
+
+
+def push_metadata_to_ipfs() -> None:
+    response = IPFSTool().client.add(
```

Comment:
Shouldn't the address be given as an arg to the script and provided here during `IPFSTool` initialization?